packet/flatcar-linux/kubernetes/ssh.tf

```diff
@@ -84,15 +83,54 @@ resource "null_resource" "bootkube-start" {
     source      = var.asset_dir
     destination = "$HOME/assets"
   }
+}
+
+// copy the calico hostendpoint controller manifests only when networking is calico
+// TODO: convert these templates to helm charts for calico host endpoint controller
+// Currently the manfifests are copied over to the calico helm chart.
+resource "null_resource" "calico-host-endpoint-manifests" {
+  count = "${var.networking == "calico" ? 1 : 0}"
+
+  depends_on = [
+    "module.bootkube",
+    "aws_route53_record.apiservers",
+    "null_resource.copy-controller-secrets",
+    "null_resource.copy-assets-dir",
+  ]
+
+  connection {
+    type    = "ssh"
+    host    = "${packet_device.controllers.0.access_public_ipv4}"
+    user    = "core"
+    timeout = "15m"
+  }
 
   provisioner "file" {
     content     = data.template_file.host_protection_policy.rendered
-    destination = "$HOME/assets/manifests-networking/calico-policy.yaml"
+    destination = "$HOME/assets/charts/kube-system/calico/templates/calico-policy.yaml"
```

Comment:
Can't we simply copy it to `manifests` directory? I'd rather not make it a part of the `calico` helm chart.

Reply:
> Can't we simply copy it to `manifests` directory? I'd rather not make it a part of the `calico` helm chart.

@invidian No we cannot  as calico-plicy.yaml depends on CRDs installed with Calico.

Any manifests found in `manifests` directory get applied first before `charts` directory. Hence the installation fails for the `calico-host-endpoint controller`.